extensions/amp-form/0.1/form-validators.js

```diff
@@ -74,6 +75,12 @@ export class FormValidator {
 
     /** @protected @const {!Document} */
     this.doc = /** @type {!Document} */ (form.ownerDocument);
+
+    /**
+     * Tribool indicating last known validity of form.
+     * @private {boolean|null}
+     */
+    this.formValidity_ = null;
```

Comment:
I think we want an event to fire on the first validation check. The null value represents "unknown" validation state.